test/src/test/java/org/axonframework/test/saga/FixtureTest_Deadlines.java

```diff
@@ -57,15 +60,26 @@ void testDeadlineScheduling() {
     @Test
     void testDeadlineSchedulingTypeMatching() {
         fixture.givenNoPriorActivity()
-               .whenAggregate("id").publishes(new TriggerSagaStartEvent("id"))
+               .whenAggregate(AGGREGATE_ID)
+               .publishes(START_SAGA_EVENT)
                .expectActiveSagas(1)
                .expectScheduledDeadlineOfType(Duration.ofMinutes(TRIGGER_DURATION_MINUTES), String.class)
                .expectNoScheduledEvents();
     }
 
+    @Test
+    void testDeadlineSchedulingNameMatching() {
+        fixture.givenAggregate(AGGREGATE_ID)
+               .published(START_SAGA_EVENT)
+               .whenAggregate(AGGREGATE_ID)
+               .publishes(new PayloadlessDeadlineShouldBeSetEvent(AGGREGATE_ID))
+               .expectScheduledDeadlineWithName(Duration.ofMinutes(TRIGGER_DURATION_MINUTES), "payloadless-deadline");
```

Comment:
Tests for the `Instant scheduledTime` option are needed as well?

Reply:
Good argument! Something I was initially thinking about as well, but decided against because the `DeadlineManagerValidator` class (used under the hood to actually perform the `expectScheduledDeadline...` methods) calls the `DeadlineManagerValidator#assertScheduledDeadlineMatching(Instant, Matcher<?>)` in the `DeadlineManagerValidator#assertScheduledDeadlineMatching(Duration, Matcher<?>)` method.

Thus, tests which validate the expect methods using `Duration`, will always end up doing a validation with `Instant`.
I am assuming this was @m1l4n54v1c initial take on only providing tests for the expect methods using `Duration` too, hence why I stuck with it too.

Fine with adding the other set though, shouldn't be that big a deal.
Or, maybe @m1l4n54v1c has a changed opinion on this part at the moment, justifying tests for the expect-with-instant methods too.